src/main/java/org/jetbrains/bsp/bazel/server/DiagnosticsDispatcher.java

```diff
@@ -67,18 +67,24 @@ public DiagnosticsDispatcher(BazelBspServer bspServer, BuildClient bspClient) {
 
   public void emitDiagnostics(
       Map<Uri, List<PublishDiagnosticsParams>> filesToDiagnostics, BuildTargetIdentifier target) {
-    for (SourceItem source : bspServer.getCachedBuildTargetSources(target)) {
-      Uri sourceUri = Uri.fromFileUri(source.getUri());
-      PublishDiagnosticsParams publishDiagnosticsParams =
-          new PublishDiagnosticsParams(
-              new TextDocumentIdentifier(sourceUri.toString()), target, new ArrayList<>(), true);
+    Streams.stream(bspServer.getCachedBuildTargetSources(target))
```

Comment:
Any reason to not call `bspServer.getCachedBuildTargetSources(target).stream()` instead of `Streams.stream()`?

Reply:
I did it because the method returned Iterable and Iterable does not have ``.stream()``, but I didn't notice it's just a method from BazelBspServer and not something external so I'll just change the return type from Iterable to Collection there.